src/components/m-accordion-item/index.js

```diff
@@ -0,0 +1,38 @@
+import wcdomready from '../../js/wcdomready';
+import { BaseComponentGlobal } from '../_abstract/component-types';
+import template from './_template';
+import styles from './index.scss';
+import AccordionItem from './js/accordion-item';
+
+class AXAAccordionItem extends BaseComponentGlobal {
+  constructor() {
+    super(styles, template);
+
+    this.selectContext('axa-accordion');
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+
+    this.className = `${this.initialClassName} m-accordion-item js-accordion-item`;
```

Comment:
@TheDadi A cool the class is already set here.

Reply:
You shouldn't need the `.js-accordion-item`. UIEvents container options is only need for web components who's container node is not the web component node itself.